vars/parseStageInfo.groovy

```diff
@@ -112,5 +112,29 @@ def call(Map config = [:]) {
     if (config['ftest_arg']) {
       result['ftest_arg'] = config['ftest_arg']
     }
+
+    result['junit_files'] = 'test_results/*.xml'
+    if (config['junit_files']) {
+      result['junit_files'] = config['junit_files']
+    }
+
+    result['artifacts'] = ['run_test.sh/*', 'vm_test/**']
+    if (config['artifacts']) {
+      result['artifacts'] = config['artifacts']
+    }
```

Comment:
Settings here are global to all stages, not just unit test stages.

This should be checking to see if this is a unit test stage from the stage name before setting a default that is specific for Unit tests.

Reply:
right, I'll add this on my next patch